solutions/cses-1640.mdx

```diff
@@ -141,3 +123,129 @@ print("IMPOSSIBLE")
 </PySection>
 
 </LanguageSection>
+
+## Solution 2 - Two Pointers
+
+By keeping two pointers, one at each end of the list, we can greedily move the left and right pointers if the sum of the two elements are less than or greater than the target sum, finding the two indices in $\mathcal{O}(N)$ time.
+<LanguageSection>
+<JavaSection>
+
+```java
```

Comment:
Also make sure to run the updated code to see if it still works properly

Reply:
yeah it got accepted 🥳